tests/strings/pybindgen/gen.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2018, Lawrence Livermore National Security, LLC. 
+# Produced at the Lawrence Livermore National Laboratory 
+# 
+# LLNL-CODE-738041.
+# All rights reserved. 
+#  
+# This file is part of Shroud.  For details, see
+# https://github.com/LLNL/shroud. Please also read shroud/LICENSE.
+#  
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are
+# met:
+#  
+# * Redistributions of source code must retain the above copyright
+#   notice, this list of conditions and the disclaimer below.
+# 
+# * Redistributions in binary form must reproduce the above copyright
+#   notice, this list of conditions and the disclaimer (as noted below)
+#   in the documentation and/or other materials provided with the
+#   distribution.
+# 
+# * Neither the name of the LLNS/LLNL nor the names of its contributors
+#   may be used to endorse or promote products derived from this
+#   software without specific prior written permission.
+# 
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+# A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL LAWRENCE
+# LIVERMORE NATIONAL SECURITY, LLC, THE U.S. DEPARTMENT OF ENERGY OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+# LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# 
+########################################################################
+"""
+Generate a module for strings using PyBindGen
```

Comment:
@ltaylor16 -- What is the connection between shroud and ``pybindgen``?

Are you just including this as an alternate python binding for the strings library, e.g. to compare against shroud's python bindings?

Reply:
I tried wrapping some of my tests using pybindgen and swig just to see what it produced.  I thought it may be useful to others to compare how to wrap stuff.  Of course, neither of those will produce a Fortran wrapper.